content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -141,7 +134,6 @@ export class ContextMenuItem extends React.PureComponent {
           tabIndex="0"
           onClick={this.onClick}
           onKeyDown={this.onKeyDown}
-          ref={option.first ? this.focusFirst : null}
```

Comment:
I rebased against a-s/master first thinking it would make it easier to update the prettier changes, I didn't realize it would add the extra commits that landed on master. I'll try to add this separately.

Reply:
you can also rebase the `whatsnew-panel` branch on master and force push.